openapi-configuration.yml

```diff
@@ -0,0 +1,29 @@
+specification: .circleci/specifications/stripe-api.yml
```

Comment:
Is this relative to the working dir or relative to the file? It should be relative to the file.